src/components/Login/components/AuthErrorsContainer/AuthErrorsContainer.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import styles from './AuthErrorsContainer.css';
+
+export default class AuthErrorsContainer extends Component {
+  static propTypes = {
+    errors: PropTypes.arrayOf(PropTypes.object).isRequired,
+  };
+
+  getErrorMessage(error = {}) {
+    const {
```

Comment:
If in Login.js you will set default value to an empty array for auth errors you will avoid default value for this method because it will newer invoked without error object